Upstream: No, Alpine specific
Reason: We don't want to log into $PWD/logs by default but
into /var/log/mautrix-meta.
diff --git a/example-config.yaml b/example-config.yaml
index a636e58..9674a1c 100644
--- a/example-config.yaml
+++ b/example-config.yaml
@@ -333,7 +333,7 @@ logging:
       format: pretty-colored
     - type: file
       format: json
-      filename: ./logs/mautrix-meta.log
+      filename: /var/log/mautrix-meta/mautrix-meta.log
       max_size: 100
       max_backups: 10
       compress: true

